src/com/homework/MyStack.java

```diff
@@ -0,0 +1,101 @@
+package com.homework;
+
+import java.util.Arrays;
+import java.util.EmptyStackException;
+
+public class MyStack<E> implements MyStackInterface<E> {
+    private final int DEFAULT_CAPACITY = 10;
+    transient Object[] elementData;
```

Comment:
Why do you use 'transient'  modifier? Explain.

Reply:
the same reason as in MyArrayList - just because in the source code of the original classes this modifier was used and I decided to leave it as is